[2024-12-26 18:48:03,298] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_two 2024-12-25T00:00:00+00:00 [queued]>
[2024-12-26 18:48:03,314] {taskinstance.py:896} INFO - Dependencies all met for <TaskInstance: assignment.task_two 2024-12-25T00:00:00+00:00 [queued]>
[2024-12-26 18:48:03,315] {taskinstance.py:1087} INFO - 
--------------------------------------------------------------------------------
[2024-12-26 18:48:03,315] {taskinstance.py:1088} INFO - Starting attempt 1 of 1
[2024-12-26 18:48:03,316] {taskinstance.py:1089} INFO - 
--------------------------------------------------------------------------------
[2024-12-26 18:48:03,323] {taskinstance.py:1107} INFO - Executing <Task(PythonOperator): task_two> on 2024-12-25T00:00:00+00:00
[2024-12-26 18:48:03,328] {standard_task_runner.py:76} INFO - Running: ['***', 'tasks', 'run', 'assignment', 'task_two', '2024-12-25T00:00:00+00:00', '--job-id', '42', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/assignment.py', '--cfg-path', '/tmp/tmpjvq0l02_', '--error-file', '/tmp/tmpl4zt_xlp']
[2024-12-26 18:48:03,326] {standard_task_runner.py:52} INFO - Started process 1641 to run task
[2024-12-26 18:48:03,330] {standard_task_runner.py:77} INFO - Job 42: Subtask task_two
[2024-12-26 18:48:03,366] {logging_mixin.py:104} INFO - Running <TaskInstance: assignment.task_two 2024-12-25T00:00:00+00:00 [running]> on host d88713c29571
[2024-12-26 18:48:03,407] {taskinstance.py:1302} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=***
AIRFLOW_CTX_DAG_ID=assignment
AIRFLOW_CTX_TASK_ID=task_two
AIRFLOW_CTX_EXECUTION_DATE=2024-12-25T00:00:00+00:00
AIRFLOW_CTX_DAG_RUN_ID=scheduled__2024-12-25T00:00:00+00:00
[2024-12-26 18:48:03,413] {python.py:151} INFO - Done. Returned value was: None
[2024-12-26 18:48:03,423] {taskinstance.py:1211} INFO - Marking task as SUCCESS. dag_id=assignment, task_id=task_two, execution_date=20241225T000000, start_date=20241226T184803, end_date=20241226T184803
[2024-12-26 18:48:03,445] {taskinstance.py:1265} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-12-26 18:48:03,464] {local_task_job.py:151} INFO - Task exited with return code 0
